pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -140,7 +140,9 @@ func (ic *ifConfigurator) configureContainerLink(
 	containerIface, err := attachContainerLink(endpoint, containerID, containerNetNS, containerIFDev)
```

Comment:
If we have check outside the func `attachContainerLink`, maybe we could remove this check: https://github.com/antrea-io/antrea/blob/main/pkg/agent/cniserver/interface_configuration_windows.go#L302

Reply:
Done.